packages/flutter_tools/templates/create/android.tmpl/settings.gradle

```diff
@@ -1 +1,15 @@
 include ':app'
+
+def flutterProjectRoot = rootProject.projectDir.parentFile.toPath()
+
+def plugins = new Properties()
+def pluginsFile = new File(flutterProjectRoot.toFile(), 'flutter-plugins.properties')
```

Comment:
Maybe change to `.flutter-plugins` to 1) make it clear you should not edit it, and 2) suggest it's generated.